Druid-Tests/DruidTestRTLCompiler.class.st

```diff
@@ -2955,6 +2955,54 @@ DruidTestRTLCompiler >> gen_primitiveDivideByConstant [
 	^ CompletePrimitive
 ]
 
+{ #category : #generated }
+DruidTestRTLCompiler >> gen_primitiveDoubleBitAndSubtreeLeftVariableLeft [
+	"AutoGenerated by Druid"
+
+	| currentBlock |
+	self MoveR: ReceiverResultReg R: ClassReg.
+	self AndCq: 16r3FFFFF R: ClassReg.
+	self MoveR: ClassReg R: ReceiverResultReg.
+	self genPrimReturn.
+	^ CompletePrimitive
+]
+
+{ #category : #generated }
+DruidTestRTLCompiler >> gen_primitiveDoubleBitAndSubtreeLeftVariableRight [
+	"AutoGenerated by Druid"
+
+	| currentBlock |
+	self MoveR: ReceiverResultReg R: ClassReg.
+	self AndCq: 16r3FFFFF R: ClassReg.
+	self MoveR: ClassReg R: ReceiverResultReg.
+	self genPrimReturn.
+	^ CompletePrimitive
+]
+
+{ #category : #generated }
+DruidTestRTLCompiler >> gen_primitiveDoubleBitAndSubtreeRightVariableLeft [
+	"AutoGenerated by Druid"
+
+	| currentBlock |
+	self MoveR: ReceiverResultReg R: ClassReg.
+	self AndCq: 16r3FFFFF R: ClassReg.
+	self MoveR: ClassReg R: ReceiverResultReg.
+	self genPrimReturn.
+	^ CompletePrimitive
+]
+
+{ #category : #generated }
+DruidTestRTLCompiler >> gen_primitiveDoubleBitAndSubtreeRightVariableRight [
+	"AutoGenerated by Druid"
+
+	| currentBlock |
+	self MoveR: ReceiverResultReg R: ClassReg.
+	self AndCq: 16r3FFFFF R: ClassReg.
+	self MoveR: ClassReg R: ReceiverResultReg.
+	self genPrimReturn.
+	^ CompletePrimitive
+]
```

Comment:
Is this right? (all the methods are equal). These are the generated, maybe it was missing to run the tests at the end?